tests/testthat/_snaps/verb-arrange.md

```diff
@@ -22,10 +22,6 @@
     Code
       # double arrange
       lf %>% arrange(a) %>% arrange(b)
-    Condition
```

Comment:
I don't think this warning was actually necessary and it just vanished.

Reply:
I think this is important, because in dplyr this would yield the equivalent of `lf %>% arrange(b, a)` because the sorts are stable.

Reply:
So should `lf %>% arrange(a) %>% arrange(b)` be equivalent to
a) `lf %>% arrange(b)` or
b) `lf %>% arrange(b, a)`?

And should it generate a warning?

The current behaviour of `dbplyr` is inconsistent

``` r
library(dplyr, warn.conflicts = FALSE)

lf <- dbplyr::lazy_frame(a = 1:3, b = 3:1)
lf %>% arrange(a) %>% arrange(b)
#> Warning: ORDER BY is ignored in subqueries without LIMIT
#> ℹ Do you need to move arrange() later in the pipeline or use window_order() instead?
#> <SQL>
#> SELECT *
#> FROM `df`
#> ORDER BY `b`

out <- dbplyr::lazy_frame(x = 1:3, y = 3:1) %>% arrange(x) %>% arrange(y)
out %>% dbplyr::op_sort()
#> [[1]]
#> <quosure>
#> expr: ^x
#> env:  0x7fb0e54abb90
#> 
#> [[2]]
#> <quosure>
#> expr: ^y
#> env:  0x7fb0e54abae8
out %>% show_query()
#> Warning: ORDER BY is ignored in subqueries without LIMIT
#> ℹ Do you need to move arrange() later in the pipeline or use window_order() instead?
#> <SQL>
#> SELECT *
#> FROM `df`
#> ORDER BY `y`
```

<sup>Created on 2022-03-04 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Reply:
I think if it returns `df %>% arrange(b, a)` then it wouldn't need a warning? But maybe it's not that important?